<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui::DiveSearch</class>
 <widget class="QWidget" name="gui::DiveSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gb_filters">
     <property name="title">
      <string>Filtres</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4" rowspan="2">
       <widget class="QCheckBox" name="cb_morning">
        <property name="text">
         <string>Matin</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_disp_start">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Du</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QCheckBox" name="cb_afternoon">
        <property name="text">
         <string>Apr√®s-midi</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="de_start"/>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="de_end"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_disp_end">
        <property name="text">
         <string>Au</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" rowspan="3">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QCheckBox" name="cb_diver_count">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Nbre de plongeurs</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QSpinBox" name="sb_diver_count"/>
      </item>
      <item row="0" column="6">
       <widget class="QCheckBox" name="cb_type">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QComboBox" name="cb_diver_count_operator">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>=</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&lt;</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="7" colspan="2">
       <widget class="QComboBox" name="combobox_type"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_status">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
